docs/configuration.md

```diff
@@ -30,6 +30,25 @@ const App = createApp({
 
 You can customize various aspects of your app by adding optional configuration:
 
+## Integrating with the Divvi Protocol
+
+Earn Divvi rewards based on the value your users bring to the blockchain protocols you integrate with. To register your app with the Divvi Protocol, follow these steps:
+
+1. Fill out the [Divvi Protocol registration form](https://docs.google.com/forms/d/e/1FAIpQLScTXExYljGoWsw4-mMHf7nXUxXV6QSrXa5zUMPK2foUwpdwZQ/viewform).
+1. Once your registration is processed, your app will be registered to the selected protocols, and Divvi rewards will start accruing based on user activity.
+
+To enable Divvi Protocol integration, add the `divviProtocol` configuration to your app:
+
+```typescript
+const App = createApp({
+  // ... basic config
+  divviProtocol: {
+    referrerId: 'someUniqueIdentifierForYourApp', // Use the same value as in the Divvi Protocol registration form
```

Comment:
Do we want this to follow a certain convention?
I think we mentioned using the same as the registry name, which is so far using PascalCase I guess.
(This reminds me I should update the example above to follow that).

Reply:
i don't have a preference so i'm happy to change this to PascalCase (i was just thinking of this as an identifier, it'll be the hex value that is used in the contract anyway)

Reply:
I'm not sure yet, but maybe we should think about the different names needed, and try to keep it simple for builders?